src/3rdparty/walkontable/src/table/master.js

```diff
@@ -35,14 +36,22 @@ class MasterTable extends Table {
 
       if (trimmingElementParent && overflow.includes(trimmingOverflow)) {
         const cloneNode = trimmingElement.cloneNode(false);
+        let cloneHeight;
 
         trimmingElementParent.insertBefore(cloneNode, trimmingElement);
```

Comment:
I think that this previous code is as good as it is. To fix an issue with `overflow: scroll` on FF I suggest to change overflow to auto before the height of the element is retrieved.
```suggestion
        cloneNode.style.overflow = 'auto';
        trimmingElementParent.insertBefore(cloneNode, trimmingElement);
```
and no more changes required.